packages/docs/content/foundation/typography/headings.mdx

```diff
@@ -184,13 +228,13 @@ Header sizes should be reduced on mobile screen sizes. Header sizes can be reduc
 - Xlarge to Large - Down 1 size
 - Medium to Small - No change in size
 
-## Typography Spacing
+### Typography Spacing
 
 Spacing between text elements can make an impact on the overall design and having a consistent spacing between elements improves the flow of a page and improves consistency between products.
 
 Here are some spacing elements that can help create guides for spacing between elements. 
 
-### How to use spacers
+#### How to use spacers
```

Comment:
Something to come back to: I don't think I've read this section before, and it's really unclear to me what we're asking Medicare teams to do with spacers. Are we asking them to place an empty div between headings and body paragraphs and add margin to it? Are we telling them to add bottom margin to headings? Top margin to body elements?

Reply:
Yeah we desperately need to work out how margin is applied to elements of the design system. Right now it's a bit messy. I think this is future discovery work https://jira.cms.gov/browse/WNMGDS-1491